### ISSUE TITLE
Max/min/size validation on list fields and list in bard fields?

### ISSUE THREAD

**Issue by JesseLeite** [statamic/v2-hub#1881]
If you try to apply `min`/`max`/`size` validation on a list field, it works, but uses an error message relevant to strings instead of arrays.  ie)

<img width="453" alt="screen shot 2018-04-09 at 5 40 23 pm" src="https://user-images.githubusercontent.com/5187394/38524590-1b90387e-3c1d-11e8-87d9-1b085c8df69c.png">

Yet, in `statamic/resources/lang/en/validation.php` there is an error message meant for arrays:

```php
"max" => [
    "numeric" => "The :attribute may not be greater than :max.",
    "file"    => "The :attribute may not be greater than :max kilobytes.",
    "string"  => "The :attribute may not be greater than :max characters.",
    "array"   => "The :attribute may not have more than :max items.",
],
```

Also, this validation works on a regular list field, but seemingly not in a list within a bard field.

**Comment by JesseLeite** [comment]
Nice, adding the `array` validation works great 👌

Do you know why these rules might not work on a list field within a bard set though?

**Comment by jasonvarga** [comment]
Yeah validation isn't supported in Replicator (+Bard) fields at the moment. #1604
